packages/buttons/src/styled/StyledButton.ts

```diff
@@ -40,6 +40,16 @@ export const getHeight = (props: IStyledButtonProps & ThemeProps<DefaultTheme>)
   return `${props.theme.space.base * 10}px`;
 };
 
+const getHorizontalPadding = (props: IStyledButtonProps & ThemeProps<DefaultTheme>) => {
+  if (props.size === SIZE.SMALL) {
+    return `${props.theme.space.base * 3}px`;
+  } else if (props.size === SIZE.LARGE) {
+    return `${props.theme.space.base * 5}px`;
+  }
+
+  return `${props.theme.space.base * 4}px`;
+};
```

Comment:
I would prefer seeing this logic added to the setup for a `padding` variable in the `sizeStyles` function rather than adding a one-off support function.

Reply:
I'm trying my best to infer patterns and I gave this change a stab. I feel like there is a lot of small things that could go different a different way so I'll be curious your thoughts.

Reply:
The `getBorderRadius` is an outlier because it is applied in the base styles where there is no space for declaring local variables. The `getHeight` is an outlier because it is exported. I'll add a comment on your improved code with a follow-on that can push the code quality just a bit further.

Reply:
Oh, just for posterity, this comment was referencing the new changes I just pushed that moved the code into `sizeStyles`, not a commentary on this.

Reply:
Nvmd I see you were also commenting on the new stuff. I'll resolve this comment.